garden.turtle.Jellybean.yml

```
@@ -0,0 +1,36 @@
+---
+app-id: garden.turtle.Jellybean
+runtime: org.gnome.Platform
+runtime-version: master
```

```suggestion
runtime-version: "45"
```

There is no master branch of runtime on Flathub

---

Oops, forgot I had that on. Changing now